servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpServerBuilder.java

```diff
@@ -330,46 +308,42 @@ public final HttpServerBuilder appendServiceFilter(final StreamingHttpServiceFil
      * @param factory {@link StreamingHttpServiceFilterFactory} to append.
      * @return {@code this}
      */
-    public final HttpServerBuilder appendServiceFilter(final Predicate<StreamingHttpRequest> predicate,
-                                                       final StreamingHttpServiceFilterFactory factory) {
-        appendServiceFilter(toConditionalServiceFilterFactory(predicate, factory));
-        return this;
-    }
+    HttpServerBuilder appendServiceFilter(Predicate<StreamingHttpRequest> predicate,
+                                                       StreamingHttpServiceFilterFactory factory);
 
     /**
      * Sets the {@link IoExecutor} to be used by this server.
      *
      * @param ioExecutor {@link IoExecutor} to use.
      * @return {@code this}.
      */
-    public abstract HttpServerBuilder ioExecutor(IoExecutor ioExecutor);
+    HttpServerBuilder ioExecutor(IoExecutor ioExecutor);
 
     /**
      * Sets the {@link Executor} to be used by this server.
      *
      * @param executor {@link Executor} to use.
      * @return {@code this}.
      */
-    public abstract HttpServerBuilder executor(Executor executor);
+    default HttpServerBuilder executor(Executor executor) {
+        throw new UnsupportedOperationException("Setting Executor not yet supported by " + getClass().getSimpleName());
```

Comment:
As described in #1819, we don't need to add default implementations for these methods back